homeassistant/components/synology_dsm/__init__.py

```diff
@@ -170,7 +178,8 @@ def _async_migrator(entity_entry: entity_registry.RegistryEntry):
     api = SynoApi(hass, entry)
     try:
         await api.async_setup()
-    except SynologyDSMRequestException as err:
+    except (SynologyDSMLoginFailedException, SynologyDSMRequestException) as err:
+        _LOGGER.debug("async_setup_entry - Unable to connect to DSM: %s", str(err))
```

Comment:
We start logging messages with capital letter.

Reply:
We don't need to copy exceptions to string when logging them. They have a `__str__` magic method already.